src/components/document.tsx

```diff
@@ -210,17 +210,20 @@ export class DocumentComponent extends BaseComponent<IProps, {}> {
 
   private renderSupportIcons() {
     const supports = this.getSupportsWithIndices();
+    const anyActive = supports.some((support) => support.item.visible);
     return (
       <div className="supports-list">
         {supports.map((support) => {
+          const {index, item} = support;
+          const visibility = !anyActive || item.visible ? "show" : "hide";
           return (
-            <span
-              key={support.index}
-              onClick={this.handleToggleSupport(support.item)}
-              className={support.item.visible ? "active" : undefined}
+            <svg
+              key={index}
+              onClick={this.handleToggleSupport(item)}
+              className={`icon icon-support${index} ${visibility}`}
```

Comment:
I might be inclined to move this into a function where (for instance) `index` can be checked to make sure that it doesn't exceed the number of CSS entries (currently four).

Reply:
Ah yes I meant to do that, good catch

Reply:
Fixed in a new commit, going to go ahead and merge